pages/design/rule-design.md

```diff
@@ -33,34 +77,74 @@ A test aspect is a part of a test subject that must be available in order to pro
 
 Other aspects may be necessary for testing. These can be added as long as they are sufficiently defined.
 
+For more details, see [ACT Rules Format: Input](https://www.w3.org/TR/act-rules-format/#input).
+
 ### Authors
 
-Names of the Authors. These must be an exact match of names in `contributors` property in `package.json`.
+Names of the Authors. These must be an exact match of names in `contributors` property in `package.json`. Note: If an author does not exist in `package.json`, they should be added to that file as part of the relevant PR.
```

Comment:
```suggestion
While optional, this can provide information on authors, previous authors, contributors, reviewers and other supporters who have helped with creation of a rule. When listing authors or other acknowledgements, names must be an exact match of names under the `contributors` property in `package.json`. Note: If an author does not exist in `package.json`, they should be added to that file as part of the relevant PR.
```